ui/guest/src/react/Guest.tsx

```diff
@@ -134,7 +135,7 @@ function Guest(props: GuestProps) {
               event.preventDefault();
               event.stopPropagation();
             }
-            dispatch({ type: type, payload: { event, record } });
+            dispatch({ type: type, payload: { event, record, rteConfig } });
```

Comment:
No need to send config here as the RTE config is already in the state.
No need for the const above either once this is removed.